src/clients/entitlements/entitlements-client.spec.ts

```diff
@@ -3,13 +3,14 @@ import { FronteggContext } from '../../components/frontegg-context';
 import { FronteggAuthenticator } from '../../authenticator';
 import { HttpClient } from '../http';
 import { mock, mockClear } from 'jest-mock-extended';
-import { VendorEntitlementsDto, VendorEntitlementsSnapshotOffsetDto } from './types';
+import { VendorEntitlementsSnapshotOffsetDto } from './types';
 import { AxiosResponse } from 'axios';
 import * as Sinon from 'sinon';
 import { useFakeTimers } from 'sinon';
 import { IUserAccessTokenWithRoles, tokenTypes } from '../identity/types';
 import { EntitlementsUserScoped } from './entitlements.user-scoped';
 import { InMemoryEntitlementsCache } from './storage/in-memory/in-memory.cache';
+import type { DTO } from '@frontegg/entitlements-service-types';
```

Comment:
this is a private package AFAIK, it won't work in the ci or for clients

Reply:
I removed the dependency and copied some types instead